examples/single-price.js

```diff
@@ -21,7 +21,7 @@ offchainOracle.getRateToEth(
         const numerator = 10 ** token.decimals;
         const denominator = 1e18; // eth decimals
         const price = parseFloat(rate) * numerator / denominator / 1e18;
-        console.log(price.toString()); // USDT in ETH
```

Comment:
This comment implies the value of USDT in the currency WETH, and in this case, the comments match what is output to the console, as price reflects how much WETH is needed for 1 USDT. Due to this misunderstanding, your comment may also be considered incorrect after some time. It seems worth expanding comments further for clarity.

Reply:
So you mean that it doesn't show how much ETH is in USDT? But the.... What else can it show?)) 
Your code console.log says 3800 eth in USDT and 0.000253747 USDT in eth. Isn't it mixed up?

Reply:
Or the comment means 1 eth is USDT 
And 1 USDT in eth?